Mayhemfile

```diff
@@ -0,0 +1,16 @@
+# Namespaced project name that the target belongs to
+project: orgize
+
+# Target name (should be unique within the project)
+target: orgize
```

Comment:
I understand that the existing fuzz target is called `fuzz_target_1`. Still, looking at the target, it looks like the target is fuzzing the Org.parse() function. Please update the target name accordingly (you may also update the name of the target if you choose, perhaps upstream would even appreciate it).